TheBTeam.ConsoleApp/TransactionViewer.cs

```diff
@@ -35,34 +41,50 @@ public static void ViewTransactionAccordingCategory(List<Transaction> transactio
             var categoryOfTransaction = ConsoleFactory.GetCategoryOfTransaction();
             Console.WriteLine($"{categoryOfTransaction}");
             var tmpTransactions = transactions.Where(t => t.User.Email == email).Where(t => t.Category == categoryOfTransaction);
-            var textPaddingWidth = 23;
+            var textPaddingWidth = 20;
             var paddingChar = ' ';
-            Console.WriteLine($"|{"Category".PadRight(textPaddingWidth, paddingChar)} " +
+            var numberOfCollumn = 5;
+
+            if (tmpTransactions.ToList().Count > 0)
+            {
+                Console.WriteLine(("").PadRight(textPaddingWidth * numberOfCollumn, '='));
```

Comment:
Why do you repeta this Code? you can use Method ViewTransaction(tmpTransactions)